File: backend/src/main/java/com/morapack/skyroute/models/Aeropuerto.java
package com.morapack.skyroute.models;

import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
@Table(name = "aeropuerto")
public class Aeropuerto {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String codigo;
    private String nombre;
    private int gmt;
    private int almacenCapacidad;
    private double latitud;
    private double longitud;
    private String continente;
    private boolean esExportador;
    private boolean flg_activo;
}

File: frontend/src/types/aeropuerto.ts
export interface Aeropuerto {
  id: number;
  codigo: string;
  nombre: string;
  gmt: number;
  almacenCapacidad: number;
  latitud: number;
  longitud: number;
  continente: string;
  esExportador: boolean;
  flg_activo: boolean;
}